xilinx/pack_io_xc7.cc

```diff
@@ -471,6 +471,7 @@ void XC7Packer::pack_io()
     hrio_rules[ctx->id("INV")].port_xform[ctx->id("O")] = ctx->id("OUT");
 
     hrio_rules[ctx->id("PS7")].new_type = ctx->id("PS7_PS7");
+	hrio_rules[ctx->id("PCIE_2_1")].new_type = ctx->id("PCIE_2_1_PCIE_2_1");
```

Comment:
Did you indent correctly here?
The indentation in the diff does not seem to match.
Can you check the indentation style (space/tabls), if it matches with the file?